specification/solutions/resource-manager/readme.md

```diff
@@ -44,6 +44,87 @@ input-file:
 - Microsoft.Solutions/stable/2021-07-01/managedapplications.json
 ```
 
+### Tag: package-managedapplications-2021-02
+
+These settings apply only when `--tag=package-managedapplications-2021-02` is specified on the command line.
+
+``` yaml $(tag) == 'package-managedapplications-2021-02'
+input-file:
+- Microsoft.Solutions/preview/2021-02-01-preview/managedapplications.json
+```
```

Comment:
tag name shall contain `preview` since it only contain files under preview folder